src/Command.js

```diff
@@ -39,7 +39,10 @@ class Command {
     static usage = '';
 
     /**
-     * commands
+     * Commands
+     * It's advised to set the main name first on
+     * the array as this is the one that will
+     * appear when using the Help command.
```

Comment:
Adding a comment here is very helpful, but I would not just make it a suggestion.
```suggestion
     * The primary command name followed by possible aliases.
     * Only the primary name is used in slash commands and the help command.
```